### PR TITLE
Tiny loader

### PR DESCRIPTION
**Pull request by justenor**

### CODE REVIEW

**Comment by lbelinsk** [comment]
build fails on linting errors

File: src/Loader/Loader.js [hunk 1]
```diff
@@ -52,6 +57,7 @@ export default class Loader extends WixComponent {
     const sizeInPx = sizesInPx[size];
     const lightArcAngle = arcsAngles[size].light;
     const darkArcAngle = arcsAngles[size].dark;
+    const supportsTooltip = size !== 'tiny'
```
Comment (lbelinsk) [inline src/Loader/Loader.js]: We are hiding here the text, so maybe a better name will be 

shouldShowText = size !== 'tiny'

**Comment by lbelinsk** [comment]
close https://github.com/wix-private/wsr-issues/issues/7

**Comment by ipanasenko** [comment]
Do you think `tiny` is okay name? We already have `x-large` and `x-small` in couple places

**Comment by lbelinsk** [comment]
Yeah, now that you mention it, you might be right.
Not sure worth changing it though, as it is already released.
We can be quick about it and change it.
@shlomitc @wuwa wdyt?

**Comment by ipanasenko** [comment]
Another solution is to deprecate it immediately and add an alias, just not to break anyone's project

**Pull request merged**